fc/jira/issue.py

```diff
@@ -116,9 +116,12 @@ def get_issue(cls, issue_id: str, auth: Auth) -> 'Issue':
         except HTTPError as exception:
             raise TaskException('Invalid issue key {}'.format(issue_id)) from exception
 
+        project = issue_json['fields']['project']['key']
         issue_type = issue_json['fields']['issuetype']['name']
-
-        if issue_type == 'Story':
+        if project != 'QPPFC':
+            issue = cls()
```

Comment:
should we change `from_json` to be a `@classmethod` like it is on other sub-classes so that this works the same way?

Reply:
Hmmm, yes, I think this would work.  Even with the subclasses calling `from_json`.